client/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    "extends": "airbnb",
+    "installedESLint": true,
+    "plugins": [
+        "react",
+        "jsx-a11y",
+        "import"
+    ],
+    "rules": {
+      "arrow-body-style": "off",
```

Comment:
If you want to exclude linter rules, you can add them individually to this object here ^